config/202-gateway.yaml

```diff
@@ -127,7 +163,8 @@ spec:
           - --discoveryAddress
           - istio-pilot:8080
           resources:
-            {}
+            requests:
+              cpu: 10m
```

Comment:
Is this an Istio change or a Knative change?

Reply:
This is an Istio change.